components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -300,6 +322,30 @@ export default class CollectionItem extends mixins(
     this.loadPriceData(data)
   }
 
+  // Get collection query with NFT Events on it
+  protected async fetchHistoryEvents() {
+    try {
+      const { data } = await this.$apollo.query<{ events: Interaction[] }>({
+        query: allCollectionSaleEvents,
+        client: 'subsquid',
+        variables: {
+          id: this.id,
+          and: {
+            // interaction_eq: 'BUY',
+          },
+        },
+      })
+      if (data && data.events && data.events.length) {
+        let events: Interaction[] = data.events
+        // TODO : default value of HISTORY for BUY
```

Comment:
Is **TODO** done?

Reply:
This TODO is not finished, another issues is created :   https://github.com/kodadot/nft-gallery/issues/2532